src/index.html

```diff
@@ -1,12 +1,419 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/src/styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="/src/images/logo.png" alt="logo">
+      </a>
+
+      <nav class="header__nav">
+        <ul class="nav-menu">
+
+          <li>
+            <a href="#" class="nav-menu__links is-active nav-menu__links--is-active">
+              Apple
+            </a>
+          </li>
```

Comment:
Fix classes everywhere
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link is-active">
              Apple
            </a>
          </li>
```